rcl_logging_spdlog/Doxyfile

```diff
@@ -0,0 +1,28 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rcl_logging_spdlog"
+PROJECT_NUMBER         = master
+PROJECT_BRIEF          = "Implementation of rcl_logging API for an spdlog backend."
+
+# Use these lines to include the generated logging_macro.h (update install path if needed)
+INPUT                  = README.md ../../../../install/rcl_logging_interface/include ./QUALITY_DECLARATION.md
```

Comment:
Why not just create a separate doxyfile for rcl_logging_interface?

Reply:
[rcl_logging_spdlog](https://github.com/ros2/rcl_logging/tree/master/rcl_logging_spdlog) doesn't have a include directory because it's using the rcl_logging_interface